.travis.yml

```diff
@@ -1,13 +1,18 @@
 language: node_js
 node_js:
-    - "0.12"
-    - "0.11"
-install:
-- npm install
+    - "stable"
+    - "4.6.0"
 services: mongodb
 env:
   global:
+  - CXX: g++-4.8
   - TWILIO_APP_SID: somenotr3al4pps1dgoeshere
   - MONGO_URL: mongodb://localhost/call-tracking
   - secure: BA1Qtq8OGfwEaXSmd20NeNGSUd4miBZe/kChnjBS2PG3JABfRITHPPT3cxwMEiNXpRRrtX+EkzwHQYgiktQBxu7CgzbSFnfmN56Xx5VrztNm6sJ0XzREvJ+fmFLLORsaRNqrRuWdOCW8j6nqr86x9ygsAaUsq1mQeOMg9YaJnlA66wxbfZ+TzHOuluGQgIAyP/HyGeqm3dLg5sNqGsvawdL5R7q5sjlYdrW/v/hjBMakLq5hi4Mf8kdalu839ZH5zPa7UOqND91gXDCAEU44txgZFuLVFrjpevoooil5Hdd+wYd3sd0b308epbcpw4bdkNRjU4h4XVwMQAomoGZQL+jbPfWgN24wk+S9e3Jg64NWOgEqGavGv6umvpzgByVNX6woLV6SDVFzGGhTmjDoQVbKOFKL+1Te/wOidrCINIQUvyPw60lKYlQgvbgnAgdwLxNxtYRwsDy9WrQOwassng7aaBGo9tLmeohe/P/6e7pBy2f6dfpR8iD9KmTmzFt3z5Z8PMjk+UCs5bNtzqcVcMGT5P9MPJw32p1qymmewuHPiKl6aOmPYAOfn9SUUFMXbvCYt6k5PkYXL4zhHg4rqnJrdMCJIa92jTuU5/YkxN8qMe0YiuFY39LYI8YC10dpKesYGIQ02edBsUVl1e2N6rCu17pQdnPVoBItYrDNvSg=
```

Comment:
I can't find out any real requests made from the test suite. Do you know why do we have this env vars (`TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN`) encrypted?